journal-letters/modernized/07-chapter-6.adoc

```diff
@@ -1,23 +1,23 @@
-== Chapter VI.
+== Chapter VI
 
-1655--Visits Scotland,
-passing through Northumberland--Has good service for the Lord
-at Edinburgh and Glasgow--Returns to Swarthmore,
-and proceeds to Cheshire.
+[.chapter-synopsis]
+* 1655--Visits Scotland, passing through Northumberland
+* Has good service for the Lord at Edinburgh and Glasgow
+* Returns to Swarthmore, and proceeds to Cheshire.
 
 It was not long after this, that I took my leave of that blessed family,
 for I was to go for Scotland, towards which nation we had felt some drawings,
 before we were clear of that service beyond the sea.
 Having taken my leave in abundance of love of my dearly beloved friends at Swarthmore,
 I went through Westmoreland, and had some good meetings among Friends there,
 which were of good service.
-Upon the 12th of the ninth month, 1655, I went towards the bishoprick,
-in order to go into Scotland.
-When I came into the bishoprick, I met with my dear companion John Stubbs again;
+Upon the 12th of the ninth month, 1655, I went towards the territory under the
+church`'s jurisdiction, in order to go into Scotland.
+When I came there, I met with my dear companion John Stubbs again;
```

Comment:
Yeah, interesting. I think I like what you've done here with `bishoprick` -- it's a pretty obsolete word, although kind of hard to replace. I'm gonna think about it a bit more, but I think this might be the way to go.